flutter/example/lib/main.dart

```diff
@@ -28,6 +28,8 @@ Future<void> main() async {
       options.considerInAppFramesByDefault = false;
       options.attachThreads = true;
       options.enableWindowMetricBreadcrumbs = true;
+      // Only usable on Flutter >= 3.3
+      options.addIntegration(OnErrorIntegration());
```

Comment:
We could maybe add a boolean in the options that control if we should use the zoned guard or the on error integration, by default, the zoned guard integration, wdyt?

Reply:
Yeah, I'll do that in a follow up. If that's fine with you?

Reply:
Sure, I'd remove this line from the sample because we're running on the latest stable version for now.